content/04.results.md

```diff
@@ -174,12 +175,38 @@ We performed gene set variant analysis (GSVA) for Hallmark cancer gene sets, dem
 
 <!-- TODO: Add immune deconvolution figure 4C information. -->
 
-#### Telomerase activity 
+![**Figure 4. Transcriptomic overview** A, First two dimensions from UMAP of sample transcriptome data.
+Points are colored by `cancer_group` of the samples they represent. B, Heatmap of GSVA scores for Hallmark gene sets with significant differences, with samples ordered by `cancer_group`. C, <!-- TODO: Add description about immune deconvolution image we use as main. -->](images/main/Figure4.png){#fig:Fig4}
+
+
+#### Prediction of _TP53_ oncogenicity and telomerase activity
+To understand the _TP53_ phenotype in each tumor, we ran a classifier previously trained on TCGA gene expression data[@doi:10.1016/j.celrep.2018.03.076] to infer _TP53_ inactivation status.
+Using high-confidence SNVs, CNVs, SVs, and fusions in _TP53_ as true positive alterations, we achieved a high accuracy (AUROC = 0.85) for rRNA-depleted, stranded samples compared to randomly shuffled _TP53_ scores (**Figure {@fig:Fig5}A**). 
+The classifier did not perform well on the poly-A samples (**Figure {@fig:FigS5}A-B**), potentially due to the low number of _TP53_ altered (N = 7) and/or total poly-A samples in our dataset (N = 58) rather than library type, as a previous study demonstrated high accuracy of this classifier on another poly-A dataset [@doi:10.1016/j.celrep.2019.09.071].
+We annotated _TP53_ alterations as "activated" if samples harbored one of p.R273C or p.R248W mutations [@doi:10.1038/ng0593-42], "loss" if the patient had a Li Fraumeni Syndrome (LFS) predisposition diagnosis or if any other SNV, CNV, SV, or fusion fell within a functional domain.
+If the _TP53_ mutation did not reside within the DNA-binding domain or if an alteration was not detected in _TP53_, we annotated the tumor as "other".
+Surprisingly, we observed that samples annotated as either "activated" or "loss" had significantly higher _TP53_ scores than those annotated as "other" (**Figure {@fig:Fig5}B**, p<sub>adj</sub> loss vs. other < 2e-16, p<sub>adj</sub> activated vs. other = 4.0e-5), suggesting that the classifier detects an oncogenic phenotype of _TP53_ (scores > 0.5) rather than solely _TP53_ inactivation.
```

Comment:
I'm not sure if this is surprising since the classifier was trained using missense variants. We _assumed_ that it'd be learning that TP53 was inactivated but in reality, it's probably safer to say altered. More precisely, it's surprising in light of the way we interpreted results in the past but not necessarily in terms of how the classifier was actually trained. This is FYI - I'm not sure if/how you want to change this sentence.

Reply:
Thanks, I'll update it